docs/resources/synthetics_concurrency_cap.md

```diff
@@ -0,0 +1,26 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_synthetics_concurrency_cap Resource - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Provides a Datadog Synthetics On Demand Concurrency Cap API resource. This can be used to manage the Concurrency Cap for Synthetics Tests.
+---
+
+# datadog_synthetics_concurrency_cap (Resource)
+
+Provides a Datadog Synthetics On Demand Concurrency Cap API resource. This can be used to manage the Concurrency Cap for Synthetics Tests.
```

Comment:
```suggestion
Provides a Datadog Synthetics On Demand Concurrency Cap API resource. This can be used to manage the Concurrency Cap for Synthetic tests.
```